src/lgdo/cli.py

```diff
@@ -5,11 +5,11 @@
 
 import lgdo
 import lgdo.logging
-from lgdo import show
+from lgdo.lh5.store import show
```

Comment:
Uhm I forgot to use relative imports in this file. Could you just import from `.` instead of `lgdo`?